pandas/core/series.py

```diff
@@ -1913,10 +1913,14 @@ def corr(self, other, method='pearson', min_periods=None):
         Parameters
         ----------
         other : Series
-        method : {'pearson', 'kendall', 'spearman'}
+        method : {'pearson', 'kendall', 'spearman'} or callable
             * pearson : standard correlation coefficient
             * kendall : Kendall Tau correlation coefficient
             * spearman : Spearman rank correlation
+            * callable: callable with input two 1d ndarray
```

Comment:
I am not sure how to doc-string this signature here

Reply:
Should I just leave it as is?

Reply:
Probably fine as is. The type would be `Callable[[ndarray, ndarray], float]`, but I'm not sure how familiar people are with typing yet.